changelog/unreleased/enhancement-sort-table-files-folders

```diff
@@ -0,0 +1,6 @@
+Enhancement: sort table files and folders
```

Comment:
The changelog item name / title suggests you're introducing sorting to the table

I'd prefer making more explicit what gets added here, e.g. "Group files and folder when sorting by name"

Reply:
yes that sounds better thx ^^